service/src/tests/mocks/mod.rs

```diff
@@ -0,0 +1,20 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+pub mod direct_request_mock;
+pub mod parentchain_api_mock;
+pub mod sidechain_api_mock;
```

Comment:
The enclave API mock should probably be added here? Or am I missing something?

Reply:
we do not use the enclave api, but somehow decided to leave the file there in case we need the tests later.

Reply:
I would still add it to the modules list here, even if it's not used at the moment. And if necessary, add the allow unused attribute to the `new` method (or whichever methods clippy complains about).

Reply:
ok.

Reply:
Yes, definitely add it. Otherwise, we don't know if it compiles.